src/modules/tracker.js

```diff
@@ -281,6 +423,71 @@ class Tracker {
     return new Error('term is a required parameter of type string');
   }
 
+  /**
+   * Send autocomplete search event to API
+   * @private
+   * @function trackSearchSubmitV2
+   * @param {string} term - Term of submitted autocomplete event
+   * @param {object} parameters - Additional parameters to be sent with request
+   * @param {string} parameters.user_input - The current autocomplete search query
+   * @param {string} [parameters.group_id] - Group identifier of selected item
+   * @param {object} [networkParameters] - Parameters relevant to the network request
+   * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
+   * @returns {(true|Error)}
+   * @description User submitted a search (pressing enter within input element, or clicking submit element)
+   * @example
+   * constructorio.tracker.trackSearchSubmitV2(
+   *     'T-Shirt',
+   *     {
+   *         user_input: 'Shirt',
+   *         group_id: '88JU230',
+   *     },
+   * );
+   */
+  trackSearchSubmitV2(term, parameters, networkParameters = {}) {
+    // Ensure term is provided (required)
+    if (term && typeof term === 'string') {
+      // Ensure parameters are provided (required)
+      if (parameters && typeof parameters === 'object' && !Array.isArray(parameters)) {
+        const baseUrl = `${this.behavioralV2Url}search_submit?`;
+        const { original_query, user_input = original_query, group_id, section } = parameters;
```

Comment:
Should we break this up into multiple lines like we do in the other functions? Same for `bodyParams` below